ports/libsimpleservo/capi/src/lib.rs

```diff
@@ -2,12 +2,29 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
+#[macro_use]
+extern crate lazy_static;
+
 #[macro_use]
 extern crate log;
 
 #[cfg(target_os = "windows")]
 mod vslogger;
 
+#[cfg(target_os = "windows")]
+use std::sync::{Arc, Mutex};
+
+#[cfg(target_os = "windows")]
+use vslogger::VsloggerModuleFilter;
+
+#[cfg(target_os = "windows")]
+lazy_static! {
+    static ref LOG_MODULE_FILTERS: Arc<Mutex<VsloggerModuleFilter>> =
```

Comment:
Need some help here:  I changed to the code to the following and also got the respective error.
```
lazy_static! {
    pub static ref LOG_MODULE_FILTERS: Arc<Mutex<vslogger_filter>> =  Arc::new(Mutex::new(::Vec<String>)); // {vslogger_filter: vec![]};
}
```

error: chained comparison operators require parentheses
  --> ports\libsimpleservo\capi\src\vslogger.rs:16:96
   |
16 |     pub static ref LOG_MODULE_FILTERS: Arc<Mutex<vslogger_filter>> =  Arc::new(Mutex::new(::Vec<String>)); // {vslogger_filter: vec![]};
   |                                                                                                ^^^^^^^^^
   |
   = help: use `::<...>` instead of `<...>` if you meant to specify type arguments
   = help: or use `(...)` if you meant to specify fn arguments

Reply:
Arc::new(Mutex::new(Vec::new()))

Reply:
or Arc::new(Mutex::new(vec![])) like you had before

Reply:
That was helpful.  I thought I tried it.  Maybe the other error got me going down a rabbit hole. 

Code:
```
// The VsloggerModuleFilter structure was replaced by below.  However, this is going away 
// since LOG_MODULE_FILTERS replaces it.  All builds now. :).  Moving on to other comments/suggestions.

lazy_static! {
    pub static ref VSLOGGER_FILTER: Vec<String> = vec![];
}

lazy_static! {
    pub static ref LOG_MODULE_FILTERS: Arc<Mutex<Vec<String>>> = Arc::new(Mutex::new(vec![]));
}
```

Reply:
Implemented.

Reply:
Had to change servo.cpp configuration of modules as such:

```
static char *pfilters[] = {…
.
.
o.vslogger_mod_size = sizeof(pfilters)/sizeof(pfilters[0]) **- 1**;
```